pyproject.toml

```diff
@@ -1,42 +1,45 @@
-[tool.poetry]
+[build-system]
+build-backend = "uv_build"
+requires = [
+  "uv-build (~=0.7.17)",
+]
+
+
+[project]
 name = "trialsynth"
-version = "0.1.0-alpha"
+version = "0.1.0a0"
 description = "Extracts clinical trial information from sources"
 authors = [
-  "tomooga <oga.t@northeastern.edu>",
-  "jim-sheldon <5619539+jim-sheldon@users.noreply.github.com>",
+  { name = "Tomo Oga", email = "oga.t@northeastern.edu" },
+  { name = "Jim Sheldon", email = "5619539+jim-sheldon@users.noreply.github.com" },
 ]
-license = "BSD-2-Clause"
 readme = "README.md"
-
-packages = [{ include = "trialsynth", from = "src" }]
-
+requires-python = ">=3.10"
+license = "BSD-2-Clause"
 classifiers = [
   "Programming Language :: Python",
   "Programming Language :: Python :: 3.10",
   "Programming Language :: Python :: 3.11",
   "Programming Language :: Python :: 3.12",
 ]
+dependencies = [
+  "addict (~=2.4)",
+  "bioregistry (~=0.11.6)",
+  "click (~=8.1)",
+  "gilda (~=1.1)",
+  "indra @ git+https://github.com/sorgerlab/indra.git",
+  "numpy (~=1.26)",  # https://github.com/allenai/scispacy/issues/536
+  "overrides (~=7.7)",
+  "pandas (~=2.2)",
+  "pydantic (~=2.7)",
```

Comment:
```suggestion
  "pydantic[email] (~=2.7)",
```

Reply:
Interesting - looks like the `email` extra was missing even before this PR. Added!